### PR TITLE
Upgrade to 1.2.3

### PR DESCRIPTION
**Pull request by Jibec**
## Problem
- new upstream versions with fixes and features

## Solution
- use latest package

## PR Status
Work finished. Package_check, basic tests and upgrade from last version OK.  
Could be reviewed and tested.

I assume the most important part is to make sure this change doesn't impact us: **Use utf8mb4 encoding for MySQL instead of utf8 (Emoji support)**

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Maniack C
- [x] **Approval (LGTM)** : Maniack C
- [x] **Approval (LGTM)** : Josué
- **CI succeeded** : [Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/view/Official_fork/job/kanboard_ynh%20PR65%20(Official_fork)/)
When the PR is mark as ready to merge, you have to wait for 3 days before really merge it.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "description": {
     "en": "Kanboard is a simple visual task board web application"
   },
-  "version": "1.2.0-0",
+  "version": "1.2.3-0",
```
Comment (Josue-T) [inline manifest.json]: Could you update the version from the last standard (probably `1.2.3~ynh1`) ?

Comment (JimboJoe) [reply manifest.json]: Bump :wink:

**Comment by maniackcrudelis** [comment]
There were many discussions about that through PR here and there, but not sure though that there's any official documentation about that. Or I didn't find it.

The idea is to add a way to know if there's an upgrade of the app itself, or only of the package.
This new format is already used by a new helper, [ynh_check_app_version_changed](https://github.com/YunoHost-Apps/Experimental_helpers/pull/17) to allow upgrade of the package without upgrading the app.

---

Will be merged in 3 days.

**Comment by maniackcrudelis** [comment]
By the way, as you have write permission in this repository, please do not fork for creating PR, create your PR from a branch in this repository.
Creating branch here will trig the dev CI, which then creating the task by itself and test the PR.

**Comment by Josue-T** [comment]
> done, where is the documentation of this new way for package version and what does this brings?

See https://github.com/YunoHost/doc/pull/626

**Pull request merged**